Assets/MRTK/Services/InputSystem/MixedRealityInputSystem.cs

```diff
@@ -239,6 +239,8 @@ public override void Initialize()
             handTrackingInputEventData = new HandTrackingInputEventData(EventSystem.current);
 
             CreateDataProviders();
+
+            IsInitialized = true;
```

Comment:
Not entirely sure why we don't call base.Initiliaze() here vs. setting the bool.

Reply:
There's a few other places in this commit where this occurs so just want to ensure consistency between manually setting vs. relying on the base class function.

Reply:
i am fine calling base.Initialize here. setting the bool was to avoid having the pattern flipped (base first, vs base last)

Reply:
So long there's consistency through the PR I'm happy with it, since it'll make it easier when writing new service classes in the future

Reply:
updated to use base.Initialize and added comments when it was "out of place"